modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java

```diff
@@ -6238,15 +6243,25 @@ private boolean processJoinRequestMessage(TcpDiscoveryJoinRequestMessage msg,
         /** */
         private IgniteProductVersion clientVer;
 
+        /** period of time after which client node which does not send heartbeat messages, is considered dead.
```

Comment:
Please check Javadoc section at
https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines
to solve codestyle issue here.